addons/escoria-core/game/core-scripts/esc/types/esc_base_command.gd

```diff
@@ -4,6 +4,17 @@ extends Node
 class_name ESCBaseCommand
 
 
+# Regex for creating command name based on the script's filename
+const COMMAND_NAME_REGEX = "(.+)\/([^.]+)(\\.[^.]*$|$)"
```

Comment:
Use a named group here.

Reply:
e.g. `(?<path>.+)\/(?<file>[^.]+)(?<extension>\\.[^.]*$|$)`

Reply:
This is still valid

Reply:
Indeed it is, and my push from yesterday addresses it. I think that's why GH marks it as "outdated".